### DISCUSSION TITLE
版本更新的双开特性？

### DISCUSSION THREAD

**Discussion by kong-que-dnf123** [Meloong-Git/PCL#2661] [讨论]
连切桌面都不能正式快照双开的特性被我解决了（？）
_（其实只是想弄明白原理uwu）_

### 重现：
1.打开最新正式版（当前为Release2.6.7[302.50]）
2.尝试打开最新快照版（当前为Snapshot 2.6.7[303.0]）
3.快照版打开失败 自动弹出挂在后台的PCL正式版
4.打开更新前拷贝的快照版（Snapshot 2.5.1）
5.旧版本快照版成功打开 实现双开
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/1bb828b2-67fa-4b34-a566-c8e60b7d4a3d)

**Comment by kong-que-dnf123** [comment]
怎么说呢版本更新的特性估计就是一个过不去的坎罢了......
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/199c0c0f-17a6-4a1f-b7b7-1de5390de634)

**Comment by LTCatt** [comment]
PCL 实际可以部分支持双开，但如果在双开的时候修改了设置，可能导致另一侧设置被覆盖或不同步，也可能因为缓存文件、日志文件冲突导致崩溃，所以为了稳定起见在一定程度上阻止了双开……
但是老版本的问题我也没法修 =。=
